src/main/java/net/snowflake/ingest/streaming/internal/BlobBuilder.java

```diff
@@ -127,7 +161,7 @@ static byte[] build(
     ByteArrayOutputStream blob = new ByteArrayOutputStream();
     if (!BLOB_NO_HEADER) {
       blob.write(BLOB_EXTENSION_TYPE.getBytes());
-      blob.write(BLOB_FORMAT_VERSION);
+      blob.write(bdecVersion.toByte());
```

Comment:
Could we pass number to server side instead of enums? I'm imagine that in the future we would have a major version and a minor version, major version will be used for major changes and minor version is for small updates, so it would be 0.0 and 1.0 in this case

Reply:
This lines passes a byte number to the server side `bdecVersion.toByte()` not enum, same as before.
Enum is now just a container for the byte number.
or did you mean something else?